src/api/instances.tsx

```diff
@@ -21,6 +21,7 @@ export const instanceEntitlements = [
   "can_delete",
   "can_edit",
   "can_exec",
+  "can_view",
```

Comment:
```suggestion
```
We never check view permission explicitly. If a user knows an entity exists, they can view it. We can always infer this permission.